src/ProductAttributePageTitle.php

```diff
@@ -7,23 +7,22 @@
  */
 class ProductAttributePageTitle {
 
-  public static function init() {
+  public static function admin_init() {
     $attribute_terms = wc_get_attribute_taxonomy_names();
     $group_filter = [];
 
-    foreach( $attribute_terms as $attribute_term ) {
-      $group_filter[] = [[
-        'param'    => 'taxonomy',
-        'operator' => '==',
-        'value'    => $attribute_term,
-      ]];
+    foreach ($attribute_terms as $attribute_term) {
+      $group_filter[] = [
+        [
+          'param' => 'taxonomy',
+          'operator' => '==',
+          'value' => $attribute_term,
```

Comment:
Are you sure that you cannot use a LIKE operator here to avoid loading all taxonomies?
```suggestion
          'operator' => 'LIKE',
          'value' => 'pa_%',
```

Reply:
I think I tried this but I will try again to be sure.

Reply:
Ok, yes I am sure.